resources/views/Widgets/Item/GraduatedPriceWidget.json

```diff
@@ -0,0 +1,18 @@
+{
+    "identifier": "Ceres::GraduatedPriceWidget",
+    "label": "Widget.graduatedPriceLabel",
+    "previewImageURL": "/images/widgets/item-image.svg",
```

Comment:
Bitte das Bild noch anpassen

Reply:
@felixdausch Passt das nun?

Reply:
nein die url passt noch nicht

Reply:
nvm passt :) in der contentWidgets.json steht es noch falsch